certbot-ci/certbot_integration_tests/certbot_tests/assertions.py

```diff
@@ -22,13 +22,13 @@
 ADMINS_SID = 'S-1-5-32-544'
 
 
-def assert_elliptic_key(key: str, curve: Type[EllipticCurve]) -> None:
+def assert_elliptic_key(key_path: str, curve: Type[EllipticCurve]) -> None:
```

Comment:
Was changing the variable name necessary to get something to pass, or just for clarity?

Reply:
There is a local variable in the function body called `key` with a different type, and it was being shadowed by the function argument. I felt that it made more sense to change the function argument.